modules/social_features/social_follow_content/social_follow_content.info.yml

```diff
@@ -3,17 +3,32 @@ description: 'Provides "Follow Content" flag type and related functionality.'
 type: module
 core: 8.x
 dependencies:
+  - activity_logger
   - activity_logger:activity_logger
+  - block
   - drupal:block
-  - dynamic_entity_reference:dynamic_entity_reference
   - drupal:field
-  - flag:flag
-  - message:message
   - drupal:node
   - drupal:options
-  - social_event:social_event
-  - social_topic:social_topic
   - drupal:system
   - drupal:user
   - drupal:views
+  - dynamic_entity_reference
+  - dynamic_entity_reference:dynamic_entity_reference
+  - field
```

Comment:
@jochemvn according to the coding standards modules need to be prefixed. E.g. Drupal core modules should be: `drupal:field`. Other modules are something like `flag:flag`.

Please fix the dependencies in this file.

See https://www.drupal.org/docs/8/creating-custom-modules/let-drupal-8-know-about-your-module-with-an-infoyml-file.